content-is-queen/creator-platform
# Reset password

This pull request overrides the default UI provided by Firebase for the reset password functionalities. The new UI is designed to enhance user experience and align with our application's branding.

**How to Test**

- Navigate to the reset password screen.

- Enter various test cases for password reset (valid email, invalid email, etc.).

- Verify that the custom UI elements display correctly.

- Ensure that success and error messages are shown appropriately.

- Check that the functionality of resetting the password works as expected with the new UI.

Closes #176 #190
